Doppler.UsersApi/Infrastructure/IAccountRepository.cs

```diff
@@ -8,5 +9,6 @@ public interface IAccountRepository
     {
         Task<ContactInformation> GetContactInformation(string accountName);
         Task UpdateContactInformation(string accountName, ContactInformation contactInformation);
+        Task<List<BasicUserInformation>> GetRelatedUsers(string email, int userType);
```

Comment:
Could be nice to add the parameter for `CancellationToken` propagation with a `default` value

```suggestion
        Task<List<BasicUserInformation>> GetRelatedUsers(string email, int userType, CancellationToken cancellationToken = default);
```